src/Game.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {RockButton} from "./RockButton";
+import {PaperButton} from "./PaperButton";
+import {ScissorsButton} from "./ScissorsButton";
+import {WaitButton} from "./WaitButton";
+import {ResultBanner} from "./ResultBanner";
+
+let houseComponent;
+
+function Game({scoreNumber, handleScoreNumber}) {
+    const [gameStatus, setGameStatus] = React.useState('initial');
+    const [buttonPicked, setButtonPicked] = React.useState('');
+    const [housePicked, setHousePicked] = React.useState('');
+    const [endGame, setEndGame] = React.useState('');
+
+
+    function handleButtonPicked(value) {
+        let userPicked = value
+        setButtonPicked(userPicked)
+        setGameStatus('user-picked')
+
+        function handleHousePicked() {
+            setHousePicked('picked');
+        }
+
+        setTimeout(handleHousePicked, 1000);
+
+        let randomNumber = Math.floor(Math.random() * (3) + 1)
+        let houseValue;
+        if (randomNumber === 1) {
+            houseValue = 'rock'
+            houseComponent = <RockButton/>
+        } else if (randomNumber === 2) {
+            houseValue = 'paper'
+            houseComponent = <PaperButton/>
+        } else {
+            houseValue = 'scissors'
+            houseComponent = <ScissorsButton/>
+        }
+
+        if (userPicked === houseValue) {
+            setEndGame('draw')
+        } else if (
+            (userPicked === 'rock' && houseValue === 'scissors') ||
+            (userPicked === 'paper' && houseValue === 'rock') ||
+            (userPicked === 'scissors' && houseValue === 'paper')
+        ) {
+            setEndGame('win');
+
+            function handleScore() {
```

Comment:
It would be best to put this function outside of `handleButtonPicked()` and receive the parameter to increase/decrease the score. i.e.:
```

function handleScore(value) { //value could be 1/-1
  handleScoreNumber(scoreNumber + value);
}
```

A better option to handle this logic could be in the `handleScoreNumber` function from the parent component.

Reply:
If I apply any of these solutions the set time out stops working

Reply:
Done